libsplinter/Cargo.toml

```diff
@@ -161,7 +162,7 @@ biome-credentials = ["bcrypt", "biome"]
 biome-key-management = ["biome"]
 biome-notifications = ["biome"]
 biome-profile = ["biome"]
-challenge-authorization = []
+challenge-authorization = ["trust-authorization"]
```

Comment:
Why is this dependency needed in this PR?

Reply:
I guess it is not needed yet, but will be as soon as the challenge authorization implementation is added. Ill remove it form this PR.

Reply:
updated